pkg/client/message.go

```diff
@@ -17,8 +17,8 @@ package client
 
 // Message is an implementation of Message interface
 type Message struct {
-	contentType  string
-	messageBody  string
-	messageCount int
-	messageID    string
+	ContentType  string
+	MessageBody  string
+	MessageCount int
```

Comment:
What is `Count`? Some documentation will help clarify it for me, and for other readers.

Reply:
it's copy paste from the widely acclaimed cli tool "stomp-tools", I'm not sure this `var` will survive to the final `Message` struct.

Currently here it will indicate the number of messages.

Reply:
Done (in a moment :-)